demo/1_generate_register_did.js

```diff
@@ -1,13 +1,13 @@
 const { PrivateKey, Client } = require("@hashgraph/sdk");
 const { HcsDid } = require("../dist");
-const { OPERATOR_ID, OPERATOR_KEY } = require("./config");
```

Comment:
this could just become 
`const { OPERATOR_ID, OPERATOR_KEY } = require("./.env.example.json");`